build.sbt

```diff
@@ -223,9 +225,7 @@ lazy val tests = project
       scalatest.value
     )
   )
-  .dependsOn(
-    cli
-  )
+  .dependsOn(coreJVM)
```

Comment:
is this equivalent?

Reply:
No, because `cli` dependent on `dynamic`